views/cart.html

```diff
@@ -13,7 +13,7 @@
     </tr>
   </thead>
   <tbody data-bind="foreach: documents">
-  <tr class="document-list" data-bind="attr: {title: displayExamType() + ', ' + displayDate() + ', ' + comment}">
+  <tr class="document-list" data-bind="attr: {title: displayExamType() + ', ' + $formatter.formatDate(date) + ', ' + comment}">
```

Comment:
Can't you push that down in the ModelView as a computed value? Setting the global is kind of ugly.